src/main/java/domain/Car.java

```diff
@@ -0,0 +1,25 @@
+package domain;
+
+import java.util.Random;
+import view.ResultView;
+
+class Car {
+    private final Position position = new Position();
+    private static final Random random = new Random();
+    private static final int MOVE_THRS = 4;
```

Comment:
기본적으로 정의를 내릴때 순서를

상수 -> 인스턴스 변수 이러한 형태로 나타냅니다.

```suggestion
    private static final Random random = new Random();
    private static final int MOVE_THRS = 4;
    private final Position position = new Position();
```